docs/content/concepts/partitions-schedules-sensors/schedules.mdx

```diff
@@ -113,9 +115,11 @@ do_stuff_partitioned_schedule = build_schedule_from_partitioned_job(
 The Partitioned Jobs concepts page also includes an [example of a date-partitioned asset](/concepts/partitions-schedules-sensors/partitions#partitioned-assets). You can define a schedule similarly using <PyObject object="build_schedule_from_partitioned_job"/>:
 
 ```python file=/concepts/partitions_schedules_sensors/schedule_from_partitions.py startafter=start_partitioned_asset_schedule endbefore=end_partitioned_asset_schedule
-from dagster import AssetGroup
+from dagster import define_asset_job
 
-partitioned_asset_job = AssetGroup([partitioned_asset]).build_job("partitioned_job")
+partitioned_asset_job = define_asset_job("partitioned_job").resolve(
```

Comment:
@OwenKephart recently added a `partitions_def` argument to `define_assets_job`, which I think we should use here instead of `resolve`